modules/openy_features/openy_node/modules/openy_node_alert/openy_node_alert.install

```diff
@@ -128,6 +130,14 @@ function openy_node_alert_update_8009() {
     'block.block.openy_lily_views_block__alerts_footer_alerts_local',
     'block.block.openy_lily_views_block__alerts_header_alerts',
     'block.block.openy_lily_views_block__alerts_header_alerts_local',
+    'block.block.openy_carnation_views_block__alerts_footer_alerts',
```

Comment:
I've tested and the problem is fixed for Open Y themes, but still actual for a custom theme (if any is used on a project).
So I guess we should check all enabled themes and remove configs according to the theme name
`block.block.{theme}_views_block__alerts_footer_alerts`, etc.

Or as an option we can look for configs with dependency to `views.view.alerts` and remove them.